go.mod

```diff
@@ -56,5 +56,7 @@ require (
 
 replace (
 	github.com/dgrijalva/jwt-go => github.com/dgrijalva/jwt-go/v4 v4.0.0-preview1
+	github.com/giantswarm/k8sclient/v7/pkg/k8scrdclient => ./pkg/k8scrdclient
```

Comment:
Why are you replacing these?

Reply:
I got confused and thought this was required because this repo imports itsefl, but its not. Good point.